lib/ruby2js/filter/functions.rb

```diff
@@ -203,8 +203,12 @@ def on_send(node)
             S(:and, s(:send, args.first, :>=, target.children.first),
               s(:send, args.first, :<, target.children.last))
           else
-            process S(:send, S(:send, target, :indexOf, args.first), :!=,
-              s(:int, -1))
+            if es2016
+              process S(:send, target, :includes, args.first)
```

Comment:
Can you mention this in the README?  Ideally in this following section: https://github.com/rubys/ruby2js/blob/dd635d5af02e72d65564f496171510cbae47aefd/README.md#es2016-support

Reply:
Done